tpot/tpot.py

```diff
@@ -39,5 +42,6 @@ class TPOTRegressor(TPOTBase):
 
     scoring_function = 'neg_mean_squared_error'  # Regression scoring
     default_config_dict = regressor_config_dict # Regression dictionary
+    lite_config_dict = regressor_config_dict_lite # Regression dictionary for TPOT lite
```

Comment:
Is storing the light config dictionary necessary here? Can't it just import and use it only when the user specifies to use TPOT light?